ci/edmtool.py

```diff
@@ -267,6 +276,61 @@ def update(runtime, toolkit, environment):
     click.echo('Done update')
 
 
+@cli.command()
+@click.option("--runtime", default="3.6", help="Python version to use")
+@click.option("--toolkit", default="null", help="Toolkit and API to use")
+@click.option("--environment", default=None, help="EDM environment to use")
+def docs(runtime, toolkit, environment):
+    """ Autogenerate documentation
+
+    """
+    parameters = get_parameters(runtime, toolkit, environment)
+    packages = " ".join(doc_dependencies)
+    ignore = " ".join(doc_ignore)
+    commands = [
+        "edm install -y -e {environment} " + packages,
+    ]
+    click.echo(
+        "Installing documentation tools in  '{environment}'".format(
+            **parameters
+        )
+    )
+    execute(commands, parameters)
+    click.echo("Done installing documentation tools")
+
+    click.echo(
+        "Regenerating API docs in  '{environment}'".format(**parameters)
+    )
+    api_path = os.path.join("docs", "source", "api")
+    if os.path.exists(api_path):
+        rmtree(api_path)
+    os.makedirs(api_path)
```

Comment:
I think this is borrowed from traitsui or pyface. Can you borrow the code from traits or traits-futures instead? It will be nice if we can borrow the sphinx api templates.